build.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
     implementation 'org.jetbrains:annotations:24.0.1'
     implementation 'io.hypersistence:hypersistence-utils-hibernate-60:3.6.1'
     implementation 'javax.persistence:javax.persistence-api:2.2'
-    implementation 'org.flywaydb:flyway-core:9.22.3'
+    implementation 'org.flywaydb:flyway-core:[9.+,10.+['
```

Comment:
@vrajmohan - The goal is to allow any latest version of `9.x`, and disallow any version of `10.x` right? 

Is it meaning full that on line 12 it is `10.0[` version here on line 78 it is `10.+[`?

Reply:
Yes, the goal is to allow any latest version of 9.x, and disallow any version of 10.x.

It is based on my reading of https://docs.gradle.org/current/userguide/single_versions.html.
> The symbol ] can be used instead of ( for an exclusive lower bound, and [ instead of ) for exclusive upper bound. e.g ]1.0, 2.0[